crane-cloud/frontend
# add pagination on admin components

# Description

This endpoint adds pagination to admin side

## Type of change

Please delete options that are not relevant.


## Trello Ticket ID

https://trello.com/c/IpRPWCWJ

## How Can This Been Tested?

Check this branch out and navigate through the different dashboards on the admin side.  Also please check thoroughly to ensure that all the functionalities on each screen still work as expected

## Checklist:


# Screenshots

![image](https://user-images.githubusercontent.com/69305400/229630198-b1e9247f-c81e-470c-99da-9cea7dcc2434.png)
